miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/UploadController.java

```diff
@@ -326,6 +328,15 @@ public void uploadLibraryList(MultipartHttpServletRequest request) throws IOExce
       log.debug("UPLOAD FAIL:", e);
     }
   }
+  
+  @RequestMapping(value = "/servicerecord", method = RequestMethod.POST)
+  public void uploadServiceRecordDocument(MultipartHttpServletRequest request) throws IOException {
+    String recordId = request.getParameter("serviceRecordId");
```

Comment:
:white_check_mark: might think about changing these from throwing exceptions to returning Bad Request objects or similar.